### PR TITLE
Second milestone: Mobile version

### PR DESCRIPTION
**Pull request by AkashaRojee**
Hello!

I used the mobile skeleton and built and styled the mobile version on top of it. Please, review the code.

The mobile skeleton contained the following:

- Navigation menu
- Headline section

The mobile version contains the following:

- Works
- About Myself

I used Grid alongside Flexbox for the positioning. I also fixed linter errors.

Please, click [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) to view the Figma design that was used.

Thanks!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,19 +23,167 @@
...
+        <div class="work-card flex-col">
+
+          <img class="flex-row" src="/images/works/work1.png" alt="Screenshot of work1">
+
+          <div class="work-info">
+            <span class="title flex-row blue-main">Tonic</span>
+            <div class="subtitle flex-row">
+              <div class="blue-light">CANOPY</div>
+              <div class="grey-main">Back-End Dev</div>
+              <div class="grey-main">2015</div>
```
Comment (Shaher-11) [inline index.html]: - There should be some space between those devs
wrong

right

File: index.html [hunk 2]
```diff
@@ -23,19 +23,167 @@
...
+
+          <img class="flex-row" src="/images/works/work2.png" alt="Screenshot of work1">
+
+          <div class="work-info">
+            <span class="title flex-row blue-main">Multi-Post Stories</span>
+            <div class="subtitle flex-row">
+              <div class="blue-light">CANOPY</div>
+              <div class="grey-main">Back-End Dev</div>
+              <div class="grey-main">2015</div>
+            </div>
```
Comment (Shaher-11) [inline index.html]: There should be some space between those devs
wrong

right

File: index.html [hunk 3]
```diff
@@ -23,19 +23,167 @@
...
+
+          <img class="flex-row" src="/images/works/work3.png" alt="Screenshot of work1">
+
+          <div class="work-info">
+            <span class="title flex-row blue-main">Multi-Post Stories</span>
+            <div class="subtitle flex-row">
+              <div class="blue-light">CANOPY</div>
+              <div class="grey-main">Back-End Dev</div>
+              <div class="grey-main">2015</div>
+            </div>
```
Comment (Shaher-11) [inline index.html]: There should be some space between those devs
wrong

right

File: index.html [hunk 4]
```diff
@@ -44,23 +192,75 @@ <h1 class="flex-row blue-main">I’m Mariane<br>Glad to see you!</h1>
...
       </section>
+
+      <!-- Skills -->
+
+      <section id="skills">
+
+        <div class="skill-card flex-col">
+
+          <div class="skill-category p-medium flex-row align-center justify-between">
+            <div>Languages</div>
```
Comment (Shaher-11) [inline index.html:230]: This text should be bold 
right

File: index.html [hunk 5]
```diff
@@ -44,23 +192,75 @@ <h1 class="flex-row blue-main">I’m Mariane<br>Glad to see you!</h1>
...
+            </div>
+          </div>
+
+        </div>
+
+        <!-- Frameworks -->
+        <div class="skill-card flex-col">
+          <div class="skill-category p-medium flex-row align-center justify-between">
+            <div>Frameworks</div>
+            <div><img src="images/arrow-collapsed.png" alt="Click here to expand"></div>
```
Comment (Shaher-11) [inline index.html:255]: This text should be bold 
right

File: index.html [hunk 6]
```diff
@@ -44,23 +192,75 @@ <h1 class="flex-row blue-main">I’m Mariane<br>Glad to see you!</h1>
...
+          </div>
+          <img src="images/horizontal-separator.png" alt="Separator">
+        </div>
+
+        <!-- Skills -->
+        <div class="skill-card flex-col">
+          <div class="skill-category p-medium flex-row align-center justify-between">
+            <div>Skills</div>
```
Comment (Shaher-11) [inline index.html]: This text should be bold 
right

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @AkashaRojee 

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] Please check the comments below for the detailed feedback.

## Optional Suggestions 💡
_The following point is  highly recommend to improve your code quality, Please consider implementing it

- Try using less ids and more classes as this good practice will save you time and repetition of your code 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

File: index.html (hunk 1 above)
Comment (AkashaRojee) [reply index.html]: I have now added an element to serve as the circle separator between the text items.

File: index.html (hunk 2 above)
Comment (AkashaRojee) [reply index.html]: I have now added an element to serve as the circle separator between the text items.

File: index.html (hunk 3 above)
Comment (AkashaRojee) [reply index.html]: I have now added an element to serve as the circle separator between the text items.

File: index.html (hunk 4 above)
Comment (AkashaRojee) [reply index.html:230]: I have now set the text to bold.

File: index.html (hunk 5 above)
Comment (AkashaRojee) [reply index.html:255]: I have now set the text to bold.

File: index.html (hunk 6 above)
Comment (AkashaRojee) [reply index.html]: I have now set the text to bold.

**Comment by AkashaRojee** [comment]
Hi @Shaher-11,

Thanks for your feedback. Here is a summary of the implemented changes as suggested:

- Add an element to serve as circle separator between text items (in individual card in Works section)
- Make text bold in Skills section

Thanks!

**Review by Shaher-11 — APPROVED** [review]
Hello @AkashaRojee ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Comment by AkashaRojee** [comment]
Thanks, @Shaher-11!

**Pull request merged**